Debug/index.html

```diff
@@ -150,18 +150,21 @@ <h2>
                                  data-position="top"
                                  data-intro="Type or edit the equation using your keyboard and the math keys below. Try using the cross out, replace, and calc buttons to help show your work.">
                                 <h3 class="sr-only">Math editor</h3>
-                                <div aria-label="edit equation" id='mathEditorActive' role="application" tabindex="0"
-                                     style="float: left; min-width: 400px; margin: 15px 0px 15px 15px;"
-                                     onfocus="TheActiveMathField.focus();"></div>
-                                <div style="float: right;">
+                                <section aria-label="edit equation" id='mathEditorActive' tabindex="0"
+                                    class="order-1"
+                                    onfocus="TheActiveMathField.focus();"
+                                    role="heading" 
+                                    aria-label="type math here"></section>
+                                <div style="float: right;" role="heading" aria-level="3">
```

Comment:
I think the 'float' property should be replaced by 'order-2' class. It will be more consistent if the position of related elements is set by the same mechanism.